.github/workflows/post-submit.yaml

```diff
@@ -8,31 +8,31 @@ on:
 
 jobs:
   push_to_registry:
-    name: Push a container image to quay.io/medik8s
+    name: Build and push images to quay.io/medik8s
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
       - name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
           go-version: 1.18
 
       - name: Log in to Quay.io
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
           registry: quay.io
 
       - name: Build and push operator and bundle images, using version 0.0.1, for pushes to main
         if: ${{ github.ref_type != 'tag' }}
-        run: export VERSION=0.0.1 && make docker-build bundle bundle-build docker-push bundle-push
+        run: make container-build container-push
 
-      - name: Build and push versioned operator and bundle images, for tags
+      - name: Build and push versioned CSV and images, for tags
         if: ${{ github.ref_type == 'tag' }}
         # remove leading 'v' from tag!
-        run: export VERSION=$(echo $GITHUB_REF_NAME | sed 's/v//') && make docker-build bundle bundle-build docker-push bundle-push
+        run: export VERSION=$(echo $GITHUB_REF_NAME | sed 's/v//') && make container-build container-push
```

Comment:
Same question as before (also not logically equivelent)

Reply:
Previously we have built and push the operator and bundle containers, and with this change we also build and push the index/catalog image. Why? It can be used for testing afterwards, and it also gives us indication whether the creation index/catalog image is successful.